src/vit-servicing-station/vit-servicing-station-lib/src/db/migrations.rs

```diff
@@ -1,20 +1,46 @@
+use tempfile::TempDir;
+
 use crate::db::DbConnection;
 
+const PG_INITIAL_SETUP_UP: &str =
+    include_str!("../../migrations/postgres/00000000000000_diesel_initial_setup/up.sql");
+const PG_SETUP_UP: &str =
+    include_str!("../../migrations/postgres/2022-11-03-210841_setup_db/up.sql");
+const SQLITE_SETUP_UP: &str =
+    include_str!("../../migrations/sqlite/2020-05-22-112032_setup_db/up.sql");
+
 pub fn initialize_db_with_migration(db_conn: &DbConnection) {
     // Discard all output
     let mut sink = std::io::sink();
 
     match db_conn {
-        DbConnection::Sqlite(db_conn) => diesel_migrations::run_pending_migrations_in_directory(
-            db_conn,
-            &std::path::PathBuf::from("./migrations/sqlite"),
-            &mut sink,
-        ),
-        DbConnection::Postgres(db_conn) => diesel_migrations::run_pending_migrations_in_directory(
-            db_conn,
-            &std::path::PathBuf::from("./migrations/postgres"),
-            &mut sink,
-        ),
+        DbConnection::Sqlite(db_conn) => {
```

Comment:
both match arms are really similar. I think they are only different in migration name and content. Maybe we can make it less duplicated?